returnn/config.py

```diff
@@ -36,6 +37,54 @@ def __repr__(self):
         return self.code
 
 
+def unparse_python(code: Any, name: Optional[str] = None, hash_full_python_code: bool = False) -> str:
```

Comment:
Why do you have that here?

Reply:
Ah sorry, I see that we are now in `returnn/config.py`.

But then my question is, why did you move this code? Why did you touch `__parse_python` at all? This is not really relevant for this PR? Or is it? How?